src/app/_utils/categoryUtils.ts

```diff
@@ -21,14 +26,9 @@ function transformCategoryDataToSettings(
   }))
 }
 
-function getCategoryData(
-  CMSCollectionKey: keyof CMSCategoryFieldMapping,
-): CMSFieldOption[] {
-  const { collection, field } = CMS_CATEGORY_FIELD_MAPPING[CMSCollectionKey]
-  const { fields } = getCollectionConfig(collection)
-  const categoriesData = getCMSFieldOptions(fields, field)
-
-  return categoriesData
+function getCategoryData(collectionName: CMSCollectionName) {
```

Comment:
I'm not sure if there's a specific need for `getCategoryData`, but I would remove it (since it's only used once) in order to reduce to cognitive load of this file.